### PR TITLE
Created Portfolio project using HTML and CSS

### PR DESCRIPTION
**Pull request by tomdan-ai**
### **MICROVERSE STUDENT PROJECT 2**

- [ ] Added README and LICENSE
- [ ] Created a pull request
- [ ] Added image folder
- [ ] Added Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          I am Tom<br />
+          Glad to see you!
+        </h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+      </div>
```
Comment (joel-kalema) [inline index.html]: - [ ] Your effort is to salute, well-done @tomdan-ai  🥳 but we are afraid to say that you have used a bad `font-family ` and `font-size` , compared to this Figma [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=2yYSS3O1IFJjyQTJ-0) you have chosen, please just one more effort, update the font-family and the font-size to match the correct template and you will be good to go ✌️

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | | headline-mobile | | mobile | |

File: README.md [hunk 2]
```diff
@@ -64,13 +64,12 @@ After you're finished please remove all the comments and instructions!
 
 > Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[your_project__name]** is a... Microverse-Student-Project-1
```
Comment (joel-kalema) [inline README.md]: I can see you have done a great job creating your readme👍 but your readme file still lacks some important features
- [ ] Make sure you removed **template logo**. You can replace it with your own logo if you have one.
- [ ] Make sure you add the project name.
- [ ] Make sure you add the project description
- [ ] Make sure you add the information about how to  getting Started on your project
      - How to install
      - How to Setup
      - How to Usage

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+.social-icons img {
+  height: 20px;
+  width: 20px;
+}
+
+.social-icons li {
+  list-style: none;
+}
+
+@media screen and (min-width: 768px) {
```
Comment (joel-kalema) [inline styles.css]: - [ ] Good job @tomdan-ai 😃 , but make sure you add also the `Minimum mobile screen size which should be 375px`. Kindly, fix it 👍

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @tomdan-ai 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Microverse-Student-Project-1</title>
+  </head>
+  <body>
+    <header>
+      <div class="nav-bar">
+        <div class="for-logo">
+          <a class="logo" href="#">Tom Daniel</a>
+        </div>
```
Comment (bobb-Rob) [inline index.html:15]: - [ ] [OPTIONAL] I noticed that your navbar logo and menu-hamburger button is not well alligned on the cross-axis. I realize that it is caused by a few CSS declarations that you have given to the `a` tag. After I removed them, it became better as you see in the second image below. 

```css
.for-logo a {
    /* width: 76px; Remove */
    /* height: 20px; Remove */
    font-family: "Poppins", sans-serif;
    font-style: normal;
    font-weight: 700;
    font-size: 18px;
    line-height: 20px;
    color: #6070ff;
    text-decoration: none;
}

.logo {
    display: flex;
    justify-content: flex-start;
    font-family: 'Poppins', sans-serif;
    font-style: normal;
    font-weight: 700;
    font-size: 18px;
    line-height: 20px;
    color: #6070ff;
    text-decoration: none;
    /* margin-top: 15px; Remove */
}
```
Your design currently
image

After I remove the styles
image

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @tomdan-ai, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**